pkg/deploy/util/util.go

```diff
@@ -14,6 +17,19 @@ import (
 	"github.com/openshift/origin/pkg/util/namer"
 )
 
+// GetDeployerPodsFor finds deployer pods using the label they should all have which correlates them to the named deployment.
+func GetDeployerPodsFor(podClient kclient.PodsNamespacer, deployment *kapi.ReplicationController) ([]kapi.Pod, error) {
```

Comment:
`DeployerPodsFor` - Get is an antiquated relic of Java and shall be burned with fire.